### PR TITLE
adding workflows and metadata files for new actions

### PR DESCRIPTION
**Pull request by dylanbannon**
### what
Adding a lot of repo bootstrap files, all of which are meant to be copied from the current repository into any other repository by use of the `auto-format.yml` workflow. Also included in the "bootstrap files" designation are the top-level `renovate.json` and `Makefile` files.

Among the bootstrap files are GitHub Actions workflows calling the following GitHub Actions:
- https://github.com/cloudposse/github-action-auto-format
- https://github.com/cloudposse/github-action-auto-release
- https://github.com/cloudposse/github-action-terraform-auto-context
- https://github.com/cloudposse/github-action-terraform-ci
- https://github.com/cloudposse/github-action-validate-codeowners

See README.md for more details.

### why
In addition to its other anointed uses, we intend to use this repo as a distribution point for Cloud Posse's suite of repo bootstrapping files in the near future. At this point, all the desired files are in the repo and their locations in the repo are identical to where they'd likely be placed in any other repo.

### CODE REVIEW

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -1 +0,0 @@
-github: cloudposse
```
Comment (osterman) [inline .github/FUNDING.yml:1]: Why are we removing this file?

Comment (osterman) [reply .github/FUNDING.yml:1]: This will turn off the advertisement to sponsor cloudposse in all of our repos

File: README.yaml [hunk 2]
```diff
@@ -31,23 +28,17 @@ badges:
 # Short description of this project
 description: |-
   ## .github
-  Community health files for the @CloudPosse organization
-
-  For more information, please see the article on [creating a default community health file for your organization](https://help.github.com/en/articles/creating-a-default-community-health-file-for-your-organization).
-
```
Comment (osterman) [inline README.yaml:37]: But this is a community health repo. https://github.com/github/.github
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

We can change the order, but we should at least acknowledge this somewhere.

File: README.yaml [hunk 3]
```diff
@@ -31,23 +28,17 @@ badges:
...
+  Standard repository backbone file for the @CloudPosse repositories. Also, some @CloudPosse org-specific files, such as `profile/README.md`.
```
Comment (osterman) [inline README.yaml:31]:
```suggestion
  Standard repository backbone file for the @cloudposse repositories. Also, some @cloudposse org-specific files, such as `profile/README.md`.
```

File: README.yaml [hunk 4]
```diff
@@ -31,23 +28,17 @@ badges:
...
-
-# How to get started quickly
-#quickstart: |-
-
-# Other files to include in this README from the project folder
-#include:
-#  - "docs/targets.md"
-#  - "docs/extensions.md"
+examples: |-
+  To add all repository backbone files to a new repo:
```
Comment (osterman) [inline README.yaml:35]:
```suggestion
  To add desired repository backbone files to a new repo:
```

File: README.yaml [hunk 5]
```diff
@@ -31,23 +28,17 @@ badges:
...
+    1. Copy everything in the `.github` folder (not repository) to the `.github folder of the destination repository.
```
Comment (osterman) [inline README.yaml:36]:
```suggestion
    1. Copy the workflows from the `.github` folder (not repository) to the `.github folder of the destination repository.
```

File: .github/FUNDING.yml (hunk 1 above)
Comment (dylanbannon) [reply .github/FUNDING.yml]: Replaced.

File: .github/workflows/terraform-ci.yml [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+on:
+  issue_comment:
+    types: [created]
+  pull_request:
+    types: [opened, synchronized, reopened]
+
+jobs:
+  terraform-ci:
+    # only run on pull requests so long as they don't come from forks
+    if: ( (github.event_name == 'pull_request') && (github.event.pull_request.head.repo.full_name == github.repository) ) || (github.event_name != 'pull_request')
```
Comment (osterman) [inline .github/workflows/terraform-ci.yml:11]: Simplify

**Review by osterman — APPROVED** [review]

File: .github/workflows/auto-format.yml [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+jobs:
+  auto-format:
+    # only run on pull requests so long as they don't come from forks
+    if: ${{ !( (github.event_name == 'pull_request') && (github.event.pull_request.head.repo.full_name != github.repository) ) }}
+    runs-on: ubuntu-latest
+    steps:
+    - uses: cloudposse/github-action-auto-format@main
+      with:
+        script-names: readme,github_format,terraform_format
```
Comment (osterman) [inline .github/workflows/auto-format.yml:17]:
```suggestion
        transformations: readme,github,terraform
```

**Review by osterman — APPROVED** [review]

**Pull request merged**